Makefile

```
@@ -308,7 +315,7 @@ bench-zk:
 
 ## server: Start API server locally (in debug mode with verbose logging) with deposit monitor and withdrawal executor
 .PHONY: server
-server: up
+server: clean-cache up build-program
```

checkmake (0.2.2)

[warning] Target body for "server" exceeds allowed length of 5 (49).

(maxbodylength)